thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/ThirdEyeAnomalyApplication.java

```diff
@@ -55,22 +49,14 @@ public String getName() {
 
   @Override
   public void initialize(final Bootstrap<ThirdEyeAnomalyConfiguration> bootstrap) {
-    bootstrap.addBundle(new MigrationsBundle<ThirdEyeAnomalyConfiguration>() {
-      @Override
```

Comment:
the last time we talked about this i got the impression that this was required for the migrations.xml. That still gets applied without this?